include/pcre2.h

```diff
@@ -89,7 +89,7 @@ will break and the relevant values must be provided by some other means. */
 
 #include <limits.h>
 #include <stdlib.h>
-#if defined(_MSC_VER) && _MSC_VER < 1700
+#if defined(_MSC_VER) && _MSC_VER <= 1700
```

Comment:
To cover VS2012 and its subversion, the check should be `_MSC_VER < 1800`. That effectively means before VS2013.